templates/pricing/desktop.html

```diff
@@ -261,32 +408,27 @@ <h2>Power to developers. Peace of mind for IT. Get Ubuntu Pro Desktop.</h2>
           </li>
           <li class="p-list__item is-ticked">Optional weekday or 24x7 support tiers</li>
         </ul>
-        <p>For more information, download our whitepaper:</p>
-        <p>
+        <div class="p-cta-block">
+          <p>For more information, download our whitepaper:</p>
           <a href="/engage/adopting-secure-enterprise-linux-desktop"
              class="p-button">Adopting a secure enterprise Linux desktop</a>
-        </p>
-        <p>
           <a href="/desktop/organisations">Find out more&nbsp;&rsaquo;</a>
-        </p>
-      </div>
-      <div class="col-5 u-align--center u-hide--medium u-hide--small">
-        {{ image (
-        url="https://assets.ubuntu.com/v1/83e7ce7c-ubuntu-pro-circular.png",
-        alt="",
-        width="250",
-        height="270",
-        hi_def=True,
-        loading="lazy"
-        ) | safe
-        }}
+        </div>
       </div>
     </div>
   </section>
 
-  {% with colour="white", bordered="false" %}
-    {% include "shared/_call-sales.html" %}
-  {% endwith %}
+  <section class="u-fixed-width">
+    <div class="p-section">
+      <hr class="p-rule" />
+      <h2 class="p-text--small-caps">Any questions?</h2>
+    </div>
+    <div class="p-section--deep">
+      <p class="p-heading--2">
+        Call us +1 737 204 0291 (Americas), +44 203 656 5291 (RoW) or <a href="/pricing/devices#get-in-touch">contact us online</a>
```

Comment:
```suggestion
        Call us +1 737 204 0291 (Americas), +44 203 656 5291 (RoW) <br />or <a href="/pricing/devices#get-in-touch">contact us online</a>
```
Adding a breakpoint to match design, can you also check against medium and small screens?